frontend/src/app/modules/scripts/scripts-page/scripts-page.component.ts

```diff
@@ -218,6 +224,18 @@ export class ScriptsPageComponent extends BaseComponent implements OnInit {
         }
     }
 
+    public scrollToResult(isSuccessful: boolean) {
+        setTimeout(() => {
+            const targetComponent = document.querySelector(
+                isSuccessful ? this.scriptResultComponentSelector : this.scriptErrorComponentSelector,
+            );
+
+            if (targetComponent) {
+                targetComponent.scrollIntoView({ behavior: 'smooth' });
+            }
+        }, 0);
```

Comment:
timeout 100 ?

Reply:
0 works as well because the main point is to do it asynchronously